### PR TITLE
Update SignalDTO to contain a list of assets to regulate

### LINKED ISSUE #130: Update SignalDTO to contain a list of assets to regulate
The SignalDTO in aFRR-NET should contain a list of assets to regulate in order to retrieve them from the prioritization service and send them to the remove control api

### PR DESCRIPTION
**Pull request by AlexStavrev**
Added AssetDTO and a list of Assets in the SignalDTO

### CODE REVIEW

File: aFRR-Service/DtoConverters/AssetDTO.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace aFRRService.DTOs;
```
Comment (mette-thorhauge) [inline aFRR-Service/DtoConverters/AssetDTO.cs:1]: Double check the namespaces of both files

**Review by mette-thorhauge — COMMENTED** [review]
Other than the possible issue with the namespace, it looks fine.

File: aFRR-Service/DtoConverters/AssetDTO.cs (hunk 1 above)
Comment (WrexBG) [reply aFRR-Service/DtoConverters/AssetDTO.cs:1]: Approved with: Commit [8e552fa](https://github.com/diversiwaver/aFRR-NET/pull/131/commits/8e552fae5090b6ce721cb81542ba109e9432eceb)

**Review by mette-thorhauge — APPROVED** [review]
Namespaces have been fixed, so I will approve the pull request

**Pull request merged**